### PR TITLE
[feature] add AddressGR

### PR DESCRIPTION
**Pull request by falegk**
Add Greek addresses

* Cities
* Regions
* Street names, numbers and street address
* zip code

### CODE REVIEW

File: lib/ffaker/address_gr.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    def city
+      CITY.sample
+    end
+
+    def street_name
+      STREET.sample
+    end
+
+    def street_nbr
+      FFaker.numerify STREET_NUMBER.sample
```
Comment (marocchino) [inline lib/ffaker/address_gr.rb]: We don't need `STREET_NUMBER.sample` here.
```ruby
FFaker.numerify "# ## ###"
```
will be fine.

File: test/test_address_gr.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# encoding: utf-8
+
+require 'helper'
+
+class TestAddressGR < Test::Unit::TestCase
+
+  def setup
+    @addressGr = FFaker::AddressGR
```
Comment (marocchino) [inline test/test_address_gr.rb]: agree with that. 
It is not common in ruby.

File: lib/ffaker/address_gr.rb (hunk 1 above)
Comment (falegk) [inline lib/ffaker/address_gr.rb]: This returns a number that looks like: "3 45 678"
But we want to generate a number up to three digits. ( 5 or 45 or 345 )
I think that it's correct :smile:

Comment (marocchino) [inline lib/ffaker/address_gr.rb]: Oh. You are right! >_<

**Comment by marocchino** [comment]
merged with 5667f13 ff9ad27 .

Thanks!

**Pull request closed without merging**